src/plugins/ab-testing-service.ts

```diff
@@ -62,6 +62,17 @@ function getEnvironment() {
   //   ABTestingEnvironment.name_of_the_test.value ||
   //   (Math.random() > 0.5 ? "A version name" : "B version name")
 
+  ABTestingEnvironment.CTA_EmailRecontact =
+    ABTestingEnvironment.CTA_EmailRecontact || {}
+  ABTestingEnvironment.CTA_EmailRecontact.index = 5
+  ABTestingEnvironment.CTA_EmailRecontact.value =
+    ABTestingEnvironment.CTA_EmailRecontact.value ||
+    (Math.random() > 0.333
+      ? Math.random() > 0.5
+        ? "version_actuelle"
+        : "version_test_1"
+      : "version_test_2")
```

Comment:
Peut être plus lisible comme ça ? 

```suggestion
const DEFAULT_CTA_EMAIL_RECONTACT = {
  index: 5,
  value: Math.random() > 0.333
    ? (Math.random() > 0.5 ? "version_actuelle" : "version_test_1")
    : "version_test_2"
};

ABTestingEnvironment.CTA_EmailRecontact = ABTestingEnvironment.CTA_EmailRecontact || DEFAULT_CTA_EMAIL_RECONTACT;
```

Reply:
J'aurais bien gardé le fait d'assigner chacun des terme et pas un objet entier comme on l'a fait avant. Cependant je peux simplifier l'expression de selection de versions